packages/migrate/src/__tests__/handlePanic.migrate.test.ts

```diff
@@ -221,11 +187,13 @@ describe('handlePanic', () => {
         draft: false,
         prismaSchema: migrate.getDatamodel(),
       })
-    } catch (err) {
-      expect(error).toMatchInlineSnapshot(`Some error message!`)
-      expect(JSON.stringify(error)).toMatchInlineSnapshot(
-        `{"rustStack":"","area":"LIFT_CLI","schemaPath":"Some Schema Path"}`,
-      )
+    } catch (error) {
+      expect(error).toMatchSnapshot()
+      expect(JSON.parse(JSON.stringify(error))).toMatchObject({
+        area: 'LIFT_CLI',
+        schemaPath,
+      })
+      expect(error.rustStack).toContain('This is the debugPanic artificial panic')
```

Comment:
How can both of this be true?

Reply:
Because Jest's API names can be misleading, `toMatchObject` actually performs exact match on a subset of properties.

Reply:
Of course 🤷